messaging/src/test/java/org/axonframework/eventhandling/deadletter/DeadLetteringEventIntegrationTest.java

```diff
@@ -438,15 +438,15 @@ void publishEventsAndProcessDeadLettersConcurrentlyShouldWorkFine() {
     }
 
     @Test
-    @Timeout(10)
+    @Timeout(20)
     void publishEventsAndProcessDeadLettersConcurrentlyInBulkShouldWorkFine() throws InterruptedException {
         int immediateSuccessesPerAggregate = 5;
         int failFirstAndThenSucceedPerAggregate = 4;
         int persistentFailingPerAggregate = 1;
         int expectedSuccessfulEvaluationCount = failFirstAndThenSucceedPerAggregate - persistentFailingPerAggregate;
         int expectedOverallSuccessfulHandlingCount = immediateSuccessesPerAggregate + expectedSuccessfulEvaluationCount;
 
-        int publishingRuns = 100;
+        int publishingRuns = 40;
```

Comment:
Yes, MySQL is very slow, so I upped the timeout as well as reduced the number of runs. I hope it will run stable on the build, or we will need to increase timeouts again.